evmcore/dummy_block.go

```diff
@@ -45,6 +45,9 @@ type (
 		GasUsed  uint64
 
 		BaseFee *big.Int
+
+		Difficulty  *big.Int
+		MixDigest   common.Hash
```

Comment:
[EIP-4399](https://eips.ethereum.org/EIPS/eip-4399) refers to this field as `mixHash` and recommends its renaming to `prevRandao`. We should probably stick to the suggested naming for those fields.